### DISCUSSION TITLE
Ghostty quit unexpectedly on M1 Pro Sequoia 15.2

### DISCUSSION THREAD

**Discussion by romdalf** [ghostty-org/ghostty#4045] [Feature Requests, Ideas] [answered]
When starting Ghostty on my M1 Pro with Sequoia 15.2, the process crashes with the following report: 

-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               ghostty [12975]
Path:                  /Volumes/*/Ghostty.app/Contents/MacOS/ghostty
Identifier:            com.mitchellh.ghostty
Version:               1.0.0 (8469)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               502

Date/Time:             2024-12-30 11:10:19.9861 +0100
OS Version:            macOS 15.2 (24C101)
Report Version:        12
Anonymous UUID:        35673463-28C0-9EB4-4A7D-C4E545D14720

Sleep/Wake UUID:       1A242A05-B47A-445E-A08F-A396D512790F

Time Awake Since Boot: 670000 seconds
Time Since Wake:       647 seconds

System Integrity Protection: enabled

Crashed Thread:        4

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000

Termination Reason:    Namespace SIGNAL, Code 6 Abort trap: 6
Terminating Process:   ghostty [12975]

Application Specific Information:
abort() called

Thread 0::  Dispatch queue: com.apple.main-thread
0   libsystem_platform.dylib      	       0x1805017bc _platform_strcmp$VARIANT$Base + 12
1   libobjc.A.dylib               	       0x180100eb4 class_getProperty + 344
2   ghostty                       	       0x100a74f10 object.Object.getProperty__anon_26849 + 36
3   ghostty                       	       0x100755e70 AppDelegate.init() + 652
4   ghostty                       	       0x100755fe0 @objc AppDelegate.init() + 20
5   AppKit                        	       0x1841628a0 -[NSClassSwapper initWithCoder:] + 744
6   UIFoundation                  	       0x18557dafc UINibDecoderDecodeObjectForValue + 684
7   UIFoundation                  	       0x18557d838 -[UINibDecoder decodeObjectForKey:] + 308
8   AppKit                        	       0x1840f18fc -[NSNibConnector initWithCoder:] + 88
9   AppKit                        	       0x1840f17b4 -[NSNibOutletConnector initWithCoder:] + 172
10  UIFoundation                  	       0x18557dafc UINibDecoderDecodeObjectForValue + 684
11  UIFoundation                  	       0x18557dc88 UINibDecoderDecodeObjectForValue + 1080
12  UIFoundation                  	       0x18557d838 -[UINibDecoder decodeObjectForKey:] + 308
13  AppKit                        	       0x1840f08f4 -[NSIBObjectData initWithCoder:] + 136
14  UIFoundation                  	       0x18557dafc UINibDecoderDecodeObjectForValue + 684
15  UIFoundation                  	       0x18557d838 -[UINibDecoder decodeObjectForKey:] + 308
16  AppKit                        	       0x1840f070c loadNib + 264
17  AppKit                        	       0x1840efd80 +[NSBundle(NSNibLoading) _loadNibFile:nameTable:options:withZone:ownerBundle:] + 560
18  AppKit                        	       0x1840efa84 -[NSBundle(NSNibLoading) loadNibNamed:owner:topLevelObjects:] + 180
19  AppKit                        	       0x1840ef8a8 +[NSBundle(NSNibLoading) loadNibNamed:owner:] + 300
20  AppKit                        	       0x1840e26cc NSApplicationMain + 496
21  ghostty                       	       0x100729990 main + 312
22  dyld                          	       0x180148274 start + 2840

Thread 1:
0   libsystem_pthread.dylib       	       0x1804c30e8 start_wqthread + 0

Thread 2:
0   libsystem_pthread.dylib       	       0x1804c30e8 start_wqthread + 0

Thread 3:
0   libsystem_pthread.dylib       	       0x1804c30e8 start_wqthread + 0

Thread 4 Crashed:
0   libsystem_kernel.dylib        	       0x18048f720 __pthread_kill + 8
1   libsystem_pthread.dylib       	       0x1804c7f70 pthread_kill + 288
2   libsystem_c.dylib             	       0x1803d4908 abort + 128
3   libsystem_malloc.dylib        	       0x1802ddea0 malloc_vreport + 896
4   libsystem_malloc.dylib        	       0x1802e1a24 malloc_report + 64
5   libsystem_malloc.dylib        	       0x180300184 find_zone_and_free + 528
6   ghostty                       	       0x100a74dcc Thread.PosixThreadImpl.spawn__anon_26771.Instance.entryFn + 284
7   libsystem_pthread.dylib       	       0x1804c30fc thread_start + 8

Thread 4 crashed with ARM Thread State (64-bit):
    x0: 0x0000000000000000   x1: 0x0000000000000000   x2: 0x0000000000000000   x3: 0x0000000000000000
    x4: 0x0000000000000073   x5: 0x000000000000002e   x6: 0x0000000000000001   x7: 0x0000000000000000
    x8: 0x8a35d8c456104819   x9: 0x8a35d8c526983819  x10: 0x000000000000000a  x11: 0x0000000000000000
   x12: 0x0000000000000031  x13: 0x0000000000000000  x14: 0x0000000000000000  x15: 0x0000000000000000
   x16: 0x0000000000000148  x17: 0x00000001f23dacb0  x18: 0x0000000000000000  x19: 0x0000000000000006
   x20: 0x0000000000007d13  x21: 0x00000001708870e0  x22: 0x000000018030d377  x23: 0x0000000170886ec0
   x24: 0x0000000000000000  x25: 0x0000000000000000  x26: 0x000000016f6dbc94  x27: 0x0000000170887000
   x28: 0x0000000000000000   fp: 0x0000000170886810   lr: 0x00000001804c7f70
    sp: 0x00000001708867f0   pc: 0x000000018048f720 cpsr: 0x40001000
   far: 0x0000000000000000  esr: 0x56000080  Address size fault

Binary Images:
       0x100724000 -        0x1016d3fff com.mitchellh.ghostty (1.0.0) <5fa411d7-5e7e-3ea2-8945-aaf65e6fafc0> /Volumes/*/Ghostty.app/Contents/MacOS/ghostty
       0x101b28000 -        0x101b6bfff org.sparkle-project.Sparkle (2.6.3) <914decbd-a6a8-366a-99ff-0316837f8814> /Volumes/*/Ghostty.app/Contents/Frameworks/Sparkle.framework/Versions/B/Sparkle
       0x1804fb000 -        0x180502ffb libsystem_platform.dylib (*) <d215e3b2-fa4f-328a-a080-7650c0991858> /usr/lib/system/libsystem_platform.dylib
       0x1800f0000 -        0x180141ca3 libobjc.A.dylib (*) <9ced21b4-d02c-31c0-bf0d-508c1a74fd93> /usr/lib/libobjc.A.dylib
       0x1840de000 -        0x185516fff com.apple.AppKit (6.9) <4251ee1c-062c-3b9a-b6e2-751f77ef9f4a> /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
       0x185517000 -        0x1856c8fff com.apple.UIFoundation (1.0) <1edae02d-93d0-3cad-80e6-55c2cd9c09ba> /System/Library/PrivateFrameworks/UIFoundation.framework/Versions/A/UIFoundation
       0x180142000 -        0x1801c3f3f dyld (*) <4ce86d18-f3fa-3d2a-a1b8-e7cd8a52fb0d> /usr/lib/dyld
       0x238711000 -        0x2387f1fff com.apple.InstalledContentLibrary (1.0) <2550dabd-ca69-3fc9-9552-902de9556d4b> /System/Library/PrivateFrameworks/InstalledContentLibrary.framework/Versions/A/InstalledContentLibrary
               0x0 - 0xffffffffffffffff ??? (*) <00000000-0000-0000-0000-000000000000> ???
       0x1804c1000 -        0x1804cdfff libsystem_pthread.dylib (*) <f1661613-b1c8-320d-b69d-97f174130e29> /usr/lib/system/libsystem_pthread.dylib
       0x180486000 -        0x1804c0ff7 libsystem_kernel.dylib (*) <dfc9530c-e75b-3d1c-a9a6-85c8a709d6a7> /usr/lib/system/libsystem_kernel.dylib
       0x18035b000 -        0x1803dcffb libsystem_c.dylib (*) <67989e50-e562-3062-9ff9-44bd90648e66> /usr/lib/system/libsystem_c.dylib
       0x1802cf000 -        0x180310ff7 libsystem_malloc.dylib (*) <d2681e0d-7831-3d34-b6bc-ce8723de184b> /usr/lib/system/libsystem_malloc.dylib

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=1.1G resident=0K(0%) swapped_out_or_unallocated=1.1G(100%)
Writable regions: Total=1.0G written=498K(0%) resident=498K(0%) swapped_out=0K(0%) unallocated=1.0G(100%)

                                VIRTUAL   REGION 
REGION TYPE                        SIZE    COUNT (non-coalesced) 
===========                     =======  ======= 
Activity Tracing                   256K        1 
ColorSync                          448K       23 
CoreGraphics                        32K        2 
CoreServices                      9472K        3 
Kernel Alloc Once                   32K        1 
MALLOC                           981.4M       44 
MALLOC guard page                  288K       18 
STACK GUARD                       56.1M        5 
Stack                             25.6M        5 
VM_ALLOCATE                         48K        3 
__AUTH                            1992K      282 
__AUTH_CONST                      30.3M      460 
__CTF                               824        1 
__DATA                            10.1M      440 
__DATA_CONST                      15.2M      466 
__DATA_DIRTY                      1029K      140 
__FONT_DATA                        2352        1 
__LINKEDIT                       609.2M        3 
__OBJC_RW                         2373K        1 
__TEXT                           567.9M      484 
__TPRO_CONST                       272K        2 
mapped file                       49.3M       12 
owned unmapped memory               32K        1 
page table in kernel               498K        1 
shared memory                      880K       14 
===========                     =======  ======= 
TOTAL                              2.3G     2413 

-----------
Full Report
-----------

{"app_name":"ghostty","timestamp":"2024-12-30 11:10:20.00 +0100","app_version":"1.0.0","slice_uuid":"5fa411d7-5e7e-3ea2-8945-aaf65e6fafc0","build_version":"8469","platform":1,"bundleID":"com.mitchellh.ghostty","share_with_app_devs":0,"is_first_party":0,"bug_type":"309","os_version":"macOS 15.2 (24C101)","roots_installed":0,"name":"ghostty","incident_id":"3C75BE57-BC4C-436C-BB51-619E4A4C8CF5"}
{
  "uptime" : 670000,
  "procRole" : "Foreground",
  "version" : 2,
  "userID" : 502,
  "deployVersion" : 210,
  "modelCode" : "MacBookPro18,1",
  "coalitionID" : 120424,
  "osVersion" : {
    "train" : "macOS 15.2",
    "build" : "24C101",
    "releaseType" : "User"
  },
  "captureTime" : "2024-12-30 11:10:19.9861 +0100",
  "codeSigningMonitor" : 1,
  "incident" : "3C75BE57-BC4C-436C-BB51-619E4A4C8CF5",
  "pid" : 12975,
  "translated" : false,
  "cpuType" : "ARM-64",
  "roots_installed" : 0,
  "bug_type" : "309",
  "procLaunch" : "2024-12-30 11:10:19.7518 +0100",
  "procStartAbsTime" : 16282925328309,
  "procExitAbsTime" : 16282930872620,
  "procName" : "ghostty",
  "procPath" : "\/Volumes\/*\/Ghostty.app\/Contents\/MacOS\/ghostty",
  "bundleInfo" : {"CFBundleShortVersionString":"1.0.0","CFBundleVersion":"8469","CFBundleIdentifier":"com.mitchellh.ghostty"},
  "storeInfo" : {"deviceIdentifierForVendor":"2DF30784-F947-542A-A387-B9512E9B3297","thirdParty":true},
  "parentProc" : "launchd",
  "parentPid" : 1,
  "coalitionName" : "com.mitchellh.ghostty",
  "crashReporterKey" : "35673463-28C0-9EB4-4A7D-C4E545D14720",
  "codeSigningID" : "com.mitchellh.ghostty",
  "codeSigningTeamID" : "24VZTF6M5V",
  "codeSigningFlags" : 570503953,
  "codeSigningValidationCategory" : 6,
  "codeSigningTrustLevel" : 4294967295,
  "instructionByteStream" : {"beforePC":"fyMD1f17v6n9AwCRcuD\/l78DAJH9e8Go\/w9f1sADX9YQKYDSARAA1A==","atPC":"AwEAVH8jA9X9e7+p\/QMAkWfg\/5e\/AwCR\/XvBqP8PX9bAA1\/WcAqA0g=="},
  "bootSessionUUID" : "4E07A2C3-2116-4240-BE68-C135BF7FD221",
  "wakeTime" : 647,
  "sleepWakeUUID" : "1A242A05-B47A-445E-A08F-A396D512790F",
  "sip" : "enabled",
  "exception" : {"codes":"0x0000000000000000, 0x0000000000000000","rawCodes":[0,0],"type":"EXC_CRASH","signal":"SIGABRT"},
  "termination" : {"flags":0,"code":6,"namespace":"SIGNAL","indicator":"Abort trap: 6","byProc":"ghostty","byPid":12975},
  "asi" : {"libsystem_c.dylib":["abort() called"]},
  "extMods" : {"caller":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"system":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"targeted":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"warnings":0},
  "faultingThread" : 4,
  "threads" : [{"id":35688203,"threadState":{"x":[{"value":4308389205,"symbolLocation":1,"symbol":"__anon_11848"},{"value":9537735936},{"value":6164424000},{"value":18446744073709551601},{"value":85},{"value":100},{"value":83},{"value":336},{"value":8393490674},{"value":6164424280},{"value":8393491304},{"value":8393491576},{"value":8393491576},{"value":2579718167717586520},{"value":1},{"value":147265848608325},{"value":6447699888,"symbolLocation":0,"symbol":"_platform_strcmp$VARIANT$Base"},{"value":8359084992},{"value":0},{"value":8218511160,"symbolLocation":0,"symbol":"OBJC_CLASS_$_NSString"},{"value":4308389204,"symbolLocation":0,"symbol":"__anon_11848"},{"value":10640589832,"symbolLocation":8,"symbol":"_OBJC_$_PROP_LIST_NSString_$_MobileInstallationAdditions"},{"value":8180380928,"symbolLocation":0,"symbol":"runtimeLock"},{"value":8218379040,"symbolLocation":224,"symbol":"_main_thread"},{"value":1081004648057274360},{"value":4306802276,"symbolLocation":0,"symbol":"io.GenericWriter(*io.counting_writer.CountingWriter(io.GenericWriter(void,error{},(function 'dummyWrite'))),error{},(function 'write')).typeErasedWriteFn"},{"value":6164424568},{"value":6164424568},{"value":12297829382473034410}],"flavor":"ARM_THREAD_STATE64","lr":{"value":6443503284},"cpsr":{"value":4096},"fp":{"value":6164424352},"sp":{"value":6164424080},"esr":{"value":2449473543,"description":"(Data Abort) byte read Translation fault"},"pc":{"value":6447699900},"far":{"value":9537735936}},"queue":"com.apple.main-thread","frames":[{"imageOffset":26556,"symbol":"_platform_strcmp$VARIANT$Base","symbolLocation":12,"imageIndex":2},{"imageOffset":69300,"symbol":"class_getProperty","symbolLocation":344,"imageIndex":3},{"imageOffset":3477264,"symbol":"object.Object.getProperty__anon_26849","symbolLocation":36,"imageIndex":0},{"imageOffset":204400,"symbol":"AppDelegate.init()","symbolLocation":652,"imageIndex":0},{"imageOffset":204768,"symbol":"@objc AppDelegate.init()","symbolLocation":20,"imageIndex":0},{"imageOffset":542880,"symbol":"-[NSClassSwapper initWithCoder:]","symbolLocation":744,"imageIndex":4},{"imageOffset":420604,"symbol":"UINibDecoderDecodeObjectForValue","symbolLocation":684,"imageIndex":5},{"imageOffset":419896,"symbol":"-[UINibDecoder decodeObjectForKey:]","symbolLocation":308,"imageIndex":5},{"imageOffset":80124,"symbol":"-[NSNibConnector initWithCoder:]","symbolLocation":88,"imageIndex":4},{"imageOffset":79796,"symbol":"-[NSNibOutletConnector initWithCoder:]","symbolLocation":172,"imageIndex":4},{"imageOffset":420604,"symbol":"UINibDecoderDecodeObjectForValue","symbolLocation":684,"imageIndex":5},{"imageOffset":421000,"symbol":"UINibDecoderDecodeObjectForValue","symbolLocation":1080,"imageIndex":5},{"imageOffset":419896,"symbol":"-[UINibDecoder decodeObjectForKey:]","symbolLocation":308,"imageIndex":5},{"imageOffset":76020,"symbol":"-[NSIBObjectData initWithCoder:]","symbolLocation":136,"imageIndex":4},{"imageOffset":420604,"symbol":"UINibDecoderDecodeObjectForValue","symbolLocation":684,"imageIndex":5},{"imageOffset":419896,"symbol":"-[UINibDecoder decodeObjectForKey:]","symbolLocation":308,"imageIndex":5},{"imageOffset":75532,"symbol":"loadNib","symbolLocation":264,"imageIndex":4},{"imageOffset":73088,"symbol":"+[NSBundle(NSNibLoading) _loadNibFile:nameTable:options:withZone:ownerBundle:]","symbolLocation":560,"imageIndex":4},{"imageOffset":72324,"symbol":"-[NSBundle(NSNibLoading) loadNibNamed:owner:topLevelObjects:]","symbolLocation":180,"imageIndex":4},{"imageOffset":71848,"symbol":"+[NSBundle(NSNibLoading) loadNibNamed:owner:]","symbolLocation":300,"imageIndex":4},{"imageOffset":18124,"symbol":"NSApplicationMain","symbolLocation":496,"imageIndex":4},{"imageOffset":22928,"symbol":"main","symbolLocation":312,"imageIndex":0},{"imageOffset":25204,"symbol":"start","symbolLocation":2840,"imageIndex":6}]},{"id":35688216,"frames":[{"imageOffset":8424,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":9}],"threadState":{"x":[{"value":6164983808},{"value":6915},{"value":6164447232},{"value":6164982656},{"value":5193730},{"value":1},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":0},"cpsr":{"value":4096},"fp":{"value":0},"sp":{"value":6164982640},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6447444200},"far":{"value":0}}},{"id":35688217,"frames":[{"imageOffset":8424,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":9}],"threadState":{"x":[{"value":6165557248},{"value":5635},{"value":6165020672},{"value":0},{"value":409602},{"value":18446744073709551615},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":0},"cpsr":{"value":4096},"fp":{"value":0},"sp":{"value":6165557248},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6447444200},"far":{"value":0}}},{"id":35688218,"frames":[{"imageOffset":8424,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":9}],"threadState":{"x":[{"value":6166130688},{"value":9219},{"value":6165594112},{"value":0},{"value":409602},{"value":18446744073709551615},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":0},"cpsr":{"value":4096},"fp":{"value":0},"sp":{"value":6166130688},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6447444200},"far":{"value":0}}},{"triggered":true,"id":35688230,"threadState":{"x":[{"value":0},{"value":0},{"value":0},{"value":0},{"value":115},{"value":46},{"value":1},{"value":0},{"value":9959104488768817177},{"value":9959104492267386905},{"value":10},{"value":0},{"value":49},{"value":0},{"value":0},{"value":0},{"value":328},{"value":8359095472},{"value":0},{"value":6},{"value":32019},{"value":6182957280},{"value":6445650807},{"value":6182956736},{"value":0},{"value":0},{"value":6164429972},{"value":6182957056},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":6447464304},"cpsr":{"value":1073745920},"fp":{"value":6182955024},"sp":{"value":6182954992},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":6447232800,"matchesCrashFrame":1},"far":{"value":0}},"frames":[{"imageOffset":38688,"symbol":"__pthread_kill","symbolLocation":8,"imageIndex":10},{"imageOffset":28528,"symbol":"pthread_kill","symbolLocation":288,"imageIndex":9},{"imageOffset":497928,"symbol":"abort","symbolLocation":128,"imageIndex":11},{"imageOffset":61088,"symbol":"malloc_vreport","symbolLocation":896,"imageIndex":12},{"imageOffset":76324,"symbol":"malloc_report","symbolLocation":64,"imageIndex":12},{"imageOffset":201092,"symbol":"find_zone_and_free","symbolLocation":528,"imageIndex":12},{"imageOffset":3476940,"symbol":"Thread.PosixThreadImpl.spawn__anon_26771.Instance.entryFn","symbolLocation":284,"imageIndex":0},{"imageOffset":8444,"symbol":"thread_start","symbolLocation":8,"imageIndex":9}]}],
  "usedImages" : [
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4302454784,
    "CFBundleShortVersionString" : "1.0.0",
    "CFBundleIdentifier" : "com.mitchellh.ghostty",
    "size" : 16449536,
    "uuid" : "5fa411d7-5e7e-3ea2-8945-aaf65e6fafc0",
    "path" : "\/Volumes\/*\/Ghostty.app\/Contents\/MacOS\/ghostty",
    "name" : "ghostty",
    "CFBundleVersion" : "8469"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4323442688,
    "CFBundleShortVersionString" : "2.6.3",
    "CFBundleIdentifier" : "org.sparkle-project.Sparkle",
    "size" : 278528,
    "uuid" : "914decbd-a6a8-366a-99ff-0316837f8814",
    "path" : "\/Volumes\/*\/Ghostty.app\/Contents\/Frameworks\/Sparkle.framework\/Versions\/B\/Sparkle",
    "name" : "Sparkle",
    "CFBundleVersion" : "2039"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6447673344,
    "size" : 32764,
    "uuid" : "d215e3b2-fa4f-328a-a080-7650c0991858",
    "path" : "\/usr\/lib\/system\/libsystem_platform.dylib",
    "name" : "libsystem_platform.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6443433984,
    "size" : 335012,
    "uuid" : "9ced21b4-d02c-31c0-bf0d-508c1a74fd93",
    "path" : "\/usr\/lib\/libobjc.A.dylib",
    "name" : "libobjc.A.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6510469120,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.AppKit",
    "size" : 21204992,
    "uuid" : "4251ee1c-062c-3b9a-b6e2-751f77ef9f4a",
    "path" : "\/System\/Library\/Frameworks\/AppKit.framework\/Versions\/C\/AppKit",
    "name" : "AppKit",
    "CFBundleVersion" : "2575.30.19"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6531674112,
    "CFBundleShortVersionString" : "1.0",
    "CFBundleIdentifier" : "com.apple.UIFoundation",
    "size" : 1777664,
    "uuid" : "1edae02d-93d0-3cad-80e6-55c2cd9c09ba",
    "path" : "\/System\/Library\/PrivateFrameworks\/UIFoundation.framework\/Versions\/A\/UIFoundation",
    "name" : "UIFoundation",
    "CFBundleVersion" : "969.5"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6443769856,
    "size" : 532288,
    "uuid" : "4ce86d18-f3fa-3d2a-a1b8-e7cd8a52fb0d",
    "path" : "\/usr\/lib\/dyld",
    "name" : "dyld"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 9536868352,
    "CFBundleShortVersionString" : "1.0",
    "CFBundleIdentifier" : "com.apple.InstalledContentLibrary",
    "size" : 921600,
    "uuid" : "2550dabd-ca69-3fc9-9552-902de9556d4b",
    "path" : "\/System\/Library\/PrivateFrameworks\/InstalledContentLibrary.framework\/Versions\/A\/InstalledContentLibrary",
    "name" : "InstalledContentLibrary",
    "CFBundleVersion" : "1.0"
  },
  {
    "size" : 0,
    "source" : "A",
    "base" : 0,
    "uuid" : "00000000-0000-0000-0000-000000000000"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6447435776,
    "size" : 53248,
    "uuid" : "f1661613-b1c8-320d-b69d-97f174130e29",
    "path" : "\/usr\/lib\/system\/libsystem_pthread.dylib",
    "name" : "libsystem_pthread.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6447194112,
    "size" : 241656,
    "uuid" : "dfc9530c-e75b-3d1c-a9a6-85c8a709d6a7",
    "path" : "\/usr\/lib\/system\/libsystem_kernel.dylib",
    "name" : "libsystem_kernel.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6445969408,
    "size" : 532476,
    "uuid" : "67989e50-e562-3062-9ff9-44bd90648e66",
    "path" : "\/usr\/lib\/system\/libsystem_c.dylib",
    "name" : "libsystem_c.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6445395968,
    "size" : 270328,
    "uuid" : "d2681e0d-7831-3d34-b6bc-ce8723de184b",
    "path" : "\/usr\/lib\/system\/libsystem_malloc.dylib",
    "name" : "libsystem_malloc.dylib"
  }
],
  "sharedCache" : {
  "base" : 6442958848,
  "size" : 4861722624,
  "uuid" : "e36100c7-89e7-3970-8938-8483ccd7019a"
},
  "vmSummary" : "ReadOnly portion of Libraries: Total=1.1G resident=0K(0%) swapped_out_or_unallocated=1.1G(100%)\nWritable regions: Total=1.0G written=498K(0%) resident=498K(0%) swapped_out=0K(0%) unallocated=1.0G(100%)\n\n                                VIRTUAL   REGION \nREGION TYPE                        SIZE    COUNT (non-coalesced) \n===========                     =======  ======= \nActivity Tracing                   256K        1 \nColorSync                          448K       23 \nCoreGraphics                        32K        2 \nCoreServices                      9472K        3 \nKernel Alloc Once                   32K        1 \nMALLOC                           981.4M       44 \nMALLOC guard page                  288K       18 \nSTACK GUARD                       56.1M        5 \nStack                             25.6M        5 \nVM_ALLOCATE                         48K        3 \n__AUTH                            1992K      282 \n__AUTH_CONST                      30.3M      460 \n__CTF                               824        1 \n__DATA                            10.1M      440 \n__DATA_CONST                      15.2M      466 \n__DATA_DIRTY                      1029K      140 \n__FONT_DATA                        2352        1 \n__LINKEDIT                       609.2M        3 \n__OBJC_RW                         2373K        1 \n__TEXT                           567.9M      484 \n__TPRO_CONST                       272K        2 \nmapped file                       49.3M       12 \nowned unmapped memory               32K        1 \npage table in kernel               498K        1 \nshared memory                      880K       14 \n===========                     =======  ======= \nTOTAL                              2.3G     2413 \n",
  "legacyInfo" : {
  "threadTriggered" : {

  }
},
  "logWritingSignature" : "cbcfe32382c0e4c3e3fa9552e2dce775e8d8001f",
  "trialInfo" : {
  "rollouts" : [
    {
      "rolloutId" : "6410af69ed1e1e7ab93ed169",
      "factorPackIds" : {

      },
      "deploymentId" : 240000011
    },
    {
      "rolloutId" : "6297d96be2c9387df974efa4",
      "factorPackIds" : {

      },
      "deploymentId" : 240000032
    }
  ],
  "experiments" : [
    {
      "treatmentId" : "45f4e2a5-551b-4bc2-a2dc-19c244dda8f8",
      "experimentId" : "6643969b3099cf28e049862f",
      "deploymentId" : 400000007
    }
  ]
}
}

Model: MacBookPro18,1, BootROM 11881.61.3, proc 10:8:2 processors, 32 GB, SMC 
Graphics: Apple M1 Pro, Apple M1 Pro, Built-In
Display: LG IPS QHD, 2560 x 1440 (QHD/WQHD - Wide Quad High Definition), Main, MirrorOff, Online
Display: LG IPS QHD, 2560 x 1440 (QHD/WQHD - Wide Quad High Definition), MirrorOff, Online
Memory Module: LPDDR5, Hynix
AirPort: spairport_wireless_card_type_wifi (0x14E4, 0x4387), wl0: Oct 31 2024 06:06:06 version 20.10.1135.4.8.7.191 FWID 01-e648b845
IO80211_driverkit-1345.8 "IO80211_driverkit-1345.8" Nov  9 2024 17:02:32
AirPort: 
Bluetooth: Version (null), 0 services, 0 devices, 0 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
USB Device: USB31Bus
USB Device: USB2.1 Hub
USB Device: HD Pro Webcam C920
USB Device: LG Monitor Controls
USB Device: USB31Bus
USB Device: USB31Bus
Thunderbolt Bus: MacBook Pro, Apple Inc.
Thunderbolt Bus: MacBook Pro, Apple Inc.
Thunderbolt Bus: MacBook Pro, Apple Inc.

**Comment by mitchellh** [comment]
There isn't enough information here. Do you have any steps to reproduce? A debug trace?

**Reply by romdalf** [reply, marked as answer]
Sorry about that. I updated it to 1.0.1, and I no longer have the crash.
